File: app/src/main/java/com/weiwa/ljl/weiwa/onAdapterEvent.java
package com.weiwa.ljl.weiwa;

/**
 * Created by hzfd on 2017/1/22.
 */
public interface onAdapterEvent {
    void onNeedInsert();
    void onNeedComment(String id);
    void onComment(String content,String id);
    void onRepost(String content,String id);
}

File: app/src/main/java/com/weiwa/ljl/weiwa/listener/onAdapterEvent.kt
package com.weiwa.ljl.weiwa.listener

/**
 * Created by hzfd on 2017/1/22.
 */
interface onAdapterEvent {
    fun onNeedInsert()
    fun onNeedComment(id: String)
    fun onComment(content: String, id: String)
    fun onRepost(content: String, id: String)
}
